.github/workflows/linux-git-devel.yml

```diff
@@ -64,11 +64,11 @@ jobs:
         run: |
           export TEST_GIT="$PWD"/git-master/git
           export TEST_GIT_EXEC_PATH=$(dirname "$TEST_GIT")
-          (cd git-branchless && cargo test --all-features)
+          (cd git-branchless && cargo test --all-features --examples --tests --workspace)
```

Comment:
Why not use `--all-targets` here so you don't need to update it if you add doc tests later?

Reply:
I tried that at first, but it also runs benchmarks, which in this case panic because the environment isn't set up.

Reply:
[`--all-targets`  doesn't run doctests](https://github.com/rust-lang/cargo/issues/6669)